ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -365,8 +365,9 @@ open class BottomCommandingController: UIViewController {
     }
 
     private func createAndBindHeroCommandView(with item: CommandingItem) -> UIView {
-        let tabItem = TabBarItem(title: item.title, image: item.image, selectedImage: item.selectedImage, largeContentImage: item.largeImage)
-        let itemView = TabBarItemView(item: tabItem, showsTitle: true)
+        let itemImage = item.image ?? UIImage()
+        let tabItem = TabBarItem(title: item.title ?? "", image: itemImage, selectedImage: item.selectedImage, largeContentImage: item.largeImage)
```

Comment:
nit: put the **item.title ?? ""** result in a constant to avoid the duplicated calls below?